LICENSE-BINARY-DIST

```diff
@@ -1,5 +1,5 @@
-LICENSE applicable to Nessie binary distributions
--------------------------------------------------
+LICENSE applicable to Nessie executables and containers
```

Comment:
Does it apply to `nessie-client` and related (integration tests) ?

Reply:
Imho, it's pretty hard to have `LICENSE` and `NOTICE` that can apply to all distributed binary artifacts. I would propose to have `LICENSE` and `NOTICE` for each distributed binary artifacts, specific for each artifact.

Reply:
It seems that we already ship correct `LICENSE`/`NOTICE`. I'm doing a full pass.